### PR TITLE
Document how to use the latest missinglink with sbt-missinglink.

### PR DESCRIPTION
**Pull request by sjrd**

### CODE REVIEW

**Review by julienrf — COMMENTED** [review]
I would rather add a comment somewhere below explaining how to use a more recent version of missinglink with sbt-missinglink, instead of having this in the “normal” setup instructions.

**Comment by sjrd** [comment]
What's wrong with it being the normal setup instructions? Having it somewhere separate suggests that wanting a more recent of missinglink is not normal and somewhat of an advanced use case. I think it's the normal thing to do, so that we don't have to update sbt-missinglink all the time.

**Comment by julienrf** [comment]
> Having it somewhere separate suggests that wanting a more recent of missinglink is not normal and somewhat of an advanced use case.

This is what I think, yes. But I am fine with your suggestion if it suits the users of sbt-missinglink. Maybe @sideeffffect want to comment?

**Comment by sideeffffect** [comment]
I think this is a good solution, second only to publishing `sbt-missinglink` on every (or most) `missinglink-core` release.
And the advantage is that it is less demanding on `sbt-missinglink` developers.
Thanks guys :+1:

File: README.md [hunk 1]
```diff
@@ -4,13 +4,17 @@ An sbt plugin for [missinglink](https://github.com/spotify/missinglink).
...
+Add the following lines in `project/plugins.sbt`:
 
 ```scala
 addSbtPlugin("ch.epfl.scala" % "sbt-missinglink" % "0.3.2")
+libraryDependencies += "com.spotify" % "missinglink-core" % "<missinglink-version>"
 ```
 
-the simply run the following task for the project you want to test:
+You can find the latest missinglink version [in their release list](https://github.com/spotify/missinglink/releases).
+As of this writing, the latest version is 0.2.5.
```
Comment (sideeffffect) [inline README.md:15]: 0.2.5 could be in
```
``
```
as in `0.2.5`. It is a super trivial thing, but it still helps.

Comment (sjrd) [reply README.md]: Done.

**Review by sideeffffect — APPROVED** [review]
Thanks! :pray:

**Review by julienrf — APPROVED** [review]

**Pull request merged**